build_a_container.rst

```diff
@@ -177,15 +177,6 @@ present on disk or in memory. See :ref:`encrypted containers
  Build options
 ***************
 
-``--builder``
-=============
-
-{Singularity} gives the option to perform a remote build. The
-``--builder`` option allows you to specify a URL to a different build
-service. For instance, you may need to specify a URL to build to an on
-premises installation of the remote builder. This option must be used in
-conjunction with ``--remote``.
-
 ``--detached``
```

Comment:
Documentation of this flag can be removed because it was only used in conjunction with `--remote` and is no longer in `apptainer`